toolkit/templates/forms/_checkbox-tree-selection-buttons.html

```diff
@@ -34,14 +34,7 @@ <h3 class="categories-heading">{{ parent_category.label }}</h3>
                 last=loop.last,
                 hint=None
               %}
-
-                {# TODO: PLS FIX #}
-                {% if PAGES_BUILDER %}
-                  {% include 'forms/_selection-button.html' %}
-                {% else %}
-                  {% include 'toolkit/forms/_selection-button.html' %}
-                {% endif %}
-
+                {% include 'toolkit/forms/_selection-button.html' %}
               {% endwith %}
```

Comment:
Do we want toolkit in front of everything?  In your main pr you suggest that we're going with the other option which would be `{% include 'forms/_selection-button.html' %}` in this case.